.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "eslint.validate": [
+      "javascript",
+      "javascriptreact",
+      "typescript",
+      "typescriptreact"
+    ],
+    "editor.codeActionsOnSave": {
+      "source.fixAll.eslint": "explicit"
+    },
+  }
```

Comment:
This is an open-source project.
I'd rather not pushing editor specific settings to the project.
Can we avoid that?

Reply:
sure.

Reply:
Removed